### ISSUE TITLE
Remove all trivial macros from Rust

### ISSUE THREAD

**Issue by dev-ardi** [rafamadriz/friendly-snippets#495]
From all of the macro snippets that exist in Rust, I think that most of them aren't very useful.

It looks like it was decided at some point that macros were hard to type...? Some of them are never used outside of macros. I'd delete all of the trivial ones and all macro attributes that don't take arguments. I'm fine with `derive`, `feature`, etc.

**Comment by keks** [comment]
I disagree on the usefulness of feature. The term "feature" is used in too many contexts for this to be useful. For example, I roughly never want to enable unstable compiler features. What I often want to do is `#[cfg(feature = "...")]`, and that feature snippet trips me up way too often.

`derive` is cool, though.
